src/scripts/OutSystems/OSUI/Utils/Network.ts

```diff
@@ -2,18 +2,22 @@
 namespace OutSystems.OSUI.Utils.Network {
 	/**
 	 * Checks if device is online
+	 *
 	 * @returns
 	 */
 	export function IsOnline(): boolean {
+		// This method can't implement the CreateApiResponse method since it's defined as a function at SS
```

Comment:
Should be:
"This method can't implement the CreateApiResponse method since it's defined as a function **in** SS"